### ISSUE TITLE
Cannot reproduce

### ISSUE THREAD

**Issue by LukeLIN-web** [allenzren/open-pi-zero#40]
The models were trained with learning rate 5e-5, global batch size 1024, and roughly 41k gradient steps with bridge (~25 epochs)   8h100  28 hours.

and 37k with fractal (~10 epochs).    4 h100 48 hours. 

But the success rate is still 1% in pick coke can and bridge evaluation .

THe fractal is this:

<img width="1780" alt="Image" src="https://github.com/user-attachments/assets/a410aff9-51e2-4720-8b60-e0b45ed5828e" /> 

The author report is here :https://wandb.ai/allenzren/train-vla/reports/Bridge-Beta--VmlldzoxMDgwODgzNQ?accessToken=8dj66y4wrgx2cjp57yfi0rplb5kc8lsxvwniuj8svsm1awt89qklfuc030foalp3

**Comment by LukeLIN-web** [comment]
For bridge I think we need more training to achieve  (  thres 0.3  )   to  0.35  ,   (thres0.,1) to 0.05    , I don't why it takes longer than readme said  https://github.com/allenzren/open-pi-zero?tab=readme-ov-file#training-details 

<img width="1694" alt="Image" src="https://github.com/user-attachments/assets/1b6c7301-f347-4411-9418-2c43022af2a8" /> 

<img width="1500" alt="Image" src="https://github.com/user-attachments/assets/ff241d8d-de6f-45f7-8044-140b7f7aa15e" />
